Formula/click.rb

```diff
@@ -1,7 +1,7 @@
 class Click < Formula
   desc "The command-line interactive controller for Kubernetes"
   homepage "https://github.com/databricks/click"
-  url "https://github.com/databricks/click/archive/v0.3.2.tar.gz"
+  url "https://github.com/databricks/click/archive/v0.4.0.tar.gz"
   sha256 "eed648409bf78a05658a9d097e5099ca17bf19df70122e2067859ae94c5575d5"
```

Comment:
Thanks for the contribution! You'll need to update the hash in addition to the URL.

Reply:
done! @mistydemeo